march_gait_selection/src/march_gait_selection/dynamic_gaits/inverted_pendulum.py

```diff
@@ -0,0 +1,38 @@
+import math
+
+
+class InvertedPendulum(object):
+    g = 9.81
```

Comment:
Class constants are usually all upper case
```suggestion
    G = 9.81
```

Reply:
Whoops